graphql-server/metadata/tables.yaml

```diff
@@ -113,6 +113,7 @@
   - role: anonymous
     permission:
       columns:
+      - host_id
```

Comment:
this permission is needed for public requests that are checking if there is a host for a game yet (to decide if to become the host); in theory, the host player id could be obfuscated by having a boolean computed column like has_host